### ISSUE TITLE
const Value cannot be stringify

### ISSUE THREAD

**Issue by GoogleCodeExporter** [byzhang/rapidjson#45]
```
Issue discovered from 
http://stackoverflow.com/questions/13200191/stringify-with-rapidjson

It is due to GenericValue::Accept() is not const. 

As GenericValue::Accept() just generates events for handler, it does not need 
to modify the value and its decedents. So it should change from:

    template <typename Handler>
    GenericValue& Accept(Handler& handler)

to

    template <typename Handler>
    const GenericValue& Accept(Handler& handler) const

```

Original issue reported on code.google.com by `milo...@gmail.com` on 14 Nov 2012 at 7:13

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision r84.
```

Original comment by `milo...@gmail.com` on 14 Nov 2012 at 7:15
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision r85.
```

Original comment by `milo...@gmail.com` on 14 Nov 2012 at 7:29

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 7606d05fedbc.
```

Original comment by `milo...@gmail.com` on 6 Jun 2014 at 6:04
